hw2/04-api.html

```diff
@@ -13,7 +13,9 @@
   <body class="bg-dark">
     <section class="container border rounded bg-light w-50 mx-auto mt-5 p-3">
       <h1 class="mt-2 mb-4">REST Countries</h1>
-      <ol id="results"></ol>
+      <ol id="results">
+       
+      </ol>
```

Comment:
Hello,

There is an empty line here. It can be removed. Are you using Prettier? It takes care of the empty lines.